extend_all.sh

```diff
@@ -1,14 +1,20 @@
 #!/bin/bash
 . util.sh
 
-if [[ $# -ne 2 ]] || [[ ! -e "$1" ]] || [[ ! -e "$2" ]] ; then
-	echo "Usage: $0 path/to/bzImage path/to/initrd"
-	exit
+# see https://www.kernel.org/doc/html/latest/x86/boot.html#details-of-harder-fileds
```

Comment:
nit: i think this is the URL you meant?

https://www.kernel.org/doc/html/latest/x86/boot.html#details-of-header-fields

Reply:
Indeed, seems it was fixed at last.